checkstyle.xml

```diff
@@ -52,6 +52,15 @@ page at http://checkstyle.sourceforge.net/config.html -->
       <property name="severity" value="error"/>
     </module>
 
+    <module name="ImportOrder">
```

Comment:
I'm not sure, but perhaps what really needs to be updated is this:

https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml

Why do we have to customize this in this project?

Reply:
"@elharo The customization was needed because we do not have formatter plugin enabled in this repository. Due to that import order are not structured properly.
 
I am planning to enable the formatter plugin after that we would not have to customize this configuration.
 
I will open a separate PR for that once that is merged will rebase this PR."